docs/introduction/09-deployment.mdx

```diff
@@ -0,0 +1,114 @@
+---
+id: deployment
+title: "Running your crawler in the Cloud"
+sidebar_label: "Deployment"
+description: Deploying Crawlee projects to the Apify Platform
+---
+
+## Apify Platform
+
+Crawlee is developed by [**Apify**](https://apify.com), the web scraping and automation platform. You could say it is the **home of Crawlee projects**. In this section we will show you how to deploy the crawler there with just a few simple steps. You can deploy a **Crawlee** project wherever you want, but using the [**Apify Platform**](https://console.apify.com) will give you the best experience.
+
+[//]: # (TODO mention the other deployment guides here once they are available)
+
+With a few simple steps, you can convert your Crawlee project into a so-called **Actor**. Actors are serverless micro-apps that are easy to develop, run, share, and integrate. The infra, proxies, and storages are ready to go. [Learn more about Actors](https://apify.com/actors).
+
+:::info
+
+We started this guide by using the Crawlee CLI to bootstrap the project - it offers the basic Crawlee templates, including a ready-made `Dockerfile`. If you know you will be deploying your project to the Apify Platform, you might want to start with the Apify CLI instead. It also offers several project templates, and those are all set up to be used on the Apify Platform right ahead.
+
+:::
+
+## Dependencies
+
+The first step will be installing two new dependencies:
+
+- Apify SDK, a toolkit for working with the Apify Platform. This will allow us to wire the storages (e.g. `RequestQueue` and `Dataset`) to the Apify cloud products. This will be a dependency of our Node.js project.
+    ```bash
+    npm install apify
+    ```
+
+- Apify CLI, a command-line tool that will help us with authentication and deployment. This will be a globally installed tool, you will install it only once and use it in all your Crawlee/Apify projects.
+    ```bash
+    npm install -g apify-cli
+    ```
+
+## Logging in to the Apify Platform
+
+The next step will be [creating your Apify account](https://console.apify.com/sign-up). Don't worry, we have a **free tier**, so you can try things out before you buy in! Once you have that, it's time to log in with the just-installed [Apify CLI](https://docs.apify.com/cli/). You will need your personal access token, which you can find at https://console.apify.com/account#/integrations.
```

Comment:
Doesn't login use the new flow that opens in a browser now, or is that still behind login-new? 👀

Reply:
Not yet, its under login-new